addons/website/static/src/builder/plugins/options/map_option.xml

```diff
@@ -55,6 +55,15 @@
                     'solid': [],
                 }"/>
     </BuilderRow>
+    <BuilderRow label.translate="Roundness" preview="false">
+        <BuilderRange
+            action="'setClassRange'"
+            actionParam="['rounded-0','rounded-1','rounded-2','rounded-3','rounded-4','rounded-5']"
+            applyTo="'.s_map .map_container'"
+            max="5"
+            step="1"
+        />
+    </BuilderRow>
```

Comment:
The **Roundness** option won’t work on older `s_map` blocks (without the `overflow-hidden` class :grimacing:). 
We can avoid migrating the old snippets by simply using CSS to calculate the inner radius of the `.s_map_embedded` (feature available since: https://github.com/odoo/odoo/commit/d0daf3990079477ef7552d769b944b55d9be4366).

You can see here an example of this for grid images : https://github.com/odoo/odoo/commit/6dc101d1b8fa586b42f9897451480ad424f5d60a.

@anso-odoo for this one, you can just remove the `overflow-hidden` classes and I’ll force-push to adapt the code (probably in a separate commit) :+1:.